drivers/include/periph/uart.h

```diff
@@ -243,6 +254,36 @@ gpio_t uart_pin_tx(uart_t uart);
 #endif /* DOXYGEN */
 #endif /* MODULE_PERIPH_UART_RECONFIGURE */
 
+#if defined(MODULE_PERIPH_UART_RX_START) || DOXYGEN
+
+/**
+ * @brief   Configure the function that will be called when a start condition
+ *          is detected.
+ *
+ *          This will not enable / disable the generation of the RX start
+ *          interrupt.
+ *
+ * @param[in] uart      The device to configure
+ * @param[in] cb        The function called when a start condition is detected
+ * @param[in] arg       Optional function argument
+ */
+void uart_rxs_configure(uart_t uart, uart_rxs_cb_t cb, void *arg);
```

Comment:
I think we should use a more verbose name here. Users stumbling over this function won't be able to tell what it does by looking at its name. Maybe something like `uart_rxstart_irq_configure()`? Maybe someone else has a better idea.

Reply:
Ok, changed the name.

Reply:
Have you omitted the `_irq` intentionally? (`uart_rxstart_irq_configure()` vs. `uart_rxstart_configure()`)

Reply:
I just wanted to keep it short, but can add the `_irq`

Reply:
Even though I never programmed Java and never experienced the common spirit of using sentences for function names, I started to prefer the descriptive names over short ones :D

But I'm okay with the short version, as well. Maybe someone else has another opinion on that?!